messages/de.json

```diff
@@ -0,0 +1,1393 @@
+        "mayor": ["Bürgermeister", "Bürgermeister"],
+        "minion": ["Untergebener", "Untergebene"],
+        "monster": ["Monster", "Monster"],
+        "mystic": ["Mystiker", "Mystiker"],
+        "oracle": ["Orakel", "Orakels"],
+        "piper": ["Dudelsackspieler", "Dudelsackspieler"],
+        "priest": ["Priester", "Priester"],
+        "prophet": ["Prophet", "Propheten"],
+        "seer": ["Seher", "Seher"],
+        "shaman": ["Shamane", "Shamanen"],
+        "sharpshooter": ["Scharfschütze", "Scharfschützen"],
+        "sorcerer": ["Hexer", "Hexer"],
+        "succubus": ["Weiblicher Dämon", "Weibliche Dämonen"],
+        "time lord": ["Zeitherr", "Zeitherren"],
+        "tough wolf": ["Harter Wolf", "Harte Wölfe"],
+        "turncoat": ["Überläufer", "Überläufer"],
+        "traitor": ["Verräter", "Verräter"],
+        "vengeful ghost": ["Rachsüchtiger Geist", "Rachsüchtige Geister"],
+        "vigilante": ["Mitglied der Bürgerwehr", "Mitglierder der Bürgerwehr"],
+        "villager": ["Bürger", "Bürger"],
+        "village drunk": ["Besoffener Bürger", "Besoffener Bürger"],
+        "warlock": ["Zauberer", "Zauberer"],
+        "werecrow": ["Wehrkrähe", "Wehrkrähe"],
+        "werekitten": ["Wehrkatze", "Wehrkatze"],
+        "wild child": ["Wildes Kind", "Wilde Kinder"],
+        "wolf": ["Wolf", "Wölfe"],
+        "wolf cub": ["Wolfsjunge", "Wolfsjungen"],
+        "wolf gunner": ["Wolf Kanonier", "Wolf Kanoniere"],
+        "wolf mystic": ["Wolf Mystiker", "Wolf Mystiker"],
+        "wolf shaman": ["Wolf Shamane", "Wolf Shamanen"],
+        "**": "The following items are names for the 'team reveal' role reveal option",
+        "village member": ["Dorfmitglied", "Dorfmitglieder"],
+        "neutral player": ["Neutraler Spieler", "Neutraler Spieler"],
+        "wolfteam player": ["Wolfteam Spieler", "Wolfteam Spieler"],
+        "***": "The following items are special keys used in player stats",
+        "lover": ["Liebhaber", "Liebhaber"],
+        "vg activated": ["vg activiert", "vgs activiert"],
+        "vg driven off": ["vg abgefahren", "vgs abgefahren"],
+        "entranced": ["verzaubert", "verzaubert"],
+        "****": "The following items are old roles that are no longer in use, but need to be here for legacy reasons",
+        "bureaucrat": ["Bürokrat", "Bürokraten"]
+    },
+    "_gamemodes": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Gamemodes",
+        "aleatoire": "zufällig",
+        "alpha": "Alpha",
+        "boreal": "Nördlich",
+        "charming": "charmant",
+        "classic": "Klassisch",
+        "default": "Standard",
+        "drunkfire": "betrunkenes Feuer",
+        "evilvillage": "böses Dorf",
+        "foolish": "töricht",
+        "guardian": "Wächter",
+        "lycan": "Lykianer",
+        "mad": "Wütend",
+        "maelstrom": "Sog",
+        "mudkip": "Mudkip",
+        "noreveal": "Noreveal",
+        "random": "zufällig",
```

Comment:
These must be unique. You re-used the same name for aleatoire and random. I suggest keeping alea in French, or picking some arbitrary other word.

Game modes don't need to be translated 1:1, you can pick something else for them instead if it makes more sense for your culture. A lot of our mode names are completely arbitrary anyway and don't even make sense in English :)